ReactSkia/pluginfactory/interfaces/RNSPlatformManagerInterface.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#pragma once
+
+#include <folly/io/async/ScopedEventBaseThread.h>
+#include <folly/Format.h>
+
+#include "ReactSkia/pluginfactory/RnsPlugin.h"
+
+namespace facebook {
+namespace react {
+namespace rnsplugin {
+
+using namespace folly;
+using namespace std;
+using folly::format;
+using std::string;
+using std::vector;
+
+class PlatformDevice;
```

Comment:
Just want ti check, Is this forward declaration needed ? As the class itself defined before it's usage.